### PR TITLE
Bumped Siddhi version

### PR DESCRIPTION
**Pull request by pcnfernando**
Siddhi Version 4.0.0-M89
Added testcases for AMRules Regressor extension

### CODE REVIEW

**Review by tishan89 — APPROVED** [review]

File: component/src/test/java/org/wso2/extension/siddhi/gpl/execution/streamingml/regression/adaptivemodelrules/AdaptiveModelRulesRegressorStreamProcessorExtensionTestcase.java [hunk 1]
```diff
@@ -350,4 +352,68 @@ public void receive(long timeStamp, Event[] inEvents, Event[] removeEvents) {
...
+
+        String inStreamDefinition = "define stream StreamA (attribute_0 double, attribute_1 double, attribute_2 "
+                + "double, attribute_3 int, attribute_4 double );";
+        String query = ("@info(name = 'query1') from StreamA#streamingml:AMRulesRegressor('ml', attribute_5 ) "
+                + "select att_0 as attribute_0, "
+                + "att_1 as attribute_1,att_2 as attribute_2,att_3 as attribute_3, accuracy insert into"
+                + " outputStream;");
+        try {
+            SiddhiAppRuntime siddhiAppRuntime = siddhiManager.createSiddhiAppRuntime(inStreamDefinition +
+                    query);
```
Comment (tishan89) [inline component/src/test/java/org/wso2/extension/siddhi/gpl/execution/streamingml/regression/adaptivemodelrules/AdaptiveModelRulesRegressorStreamProcessorExtensionTestcase.java:370]: Need an Assert.fail() here

File: component/src/test/java/org/wso2/extension/siddhi/gpl/execution/streamingml/regression/adaptivemodelrules/AdaptiveModelRulesRegressorStreamProcessorExtensionTestcase.java [hunk 2]
```diff
@@ -350,4 +352,68 @@ public void receive(long timeStamp, Event[] inEvents, Event[] removeEvents) {
...
+            inputHandler.send(new Object[]{14.6, 39.31, 1011.11, 72.52, 464.16});
+            inputHandler.send(new Object[]{7.91, 39.96, 1023.57, 88.44, 475.52});
+            inputHandler.send(new Object[]{5.81, 35.79, 1012.14, 92.28, 484.41});
+            inputHandler.send(new Object[]{30.53, 65.18, 1012.69, 41.85, 437.89});
+
+            Thread.sleep(1100);
+
+            InputHandler inputHandler1 = siddhiAppRuntime.getInputHandler("StreamA");
+            // send some unseen data for prediction
+            inputHandler1.send(new Object[]{14.96, "string", 1024.07, 73.17});
```
Comment (tishan89) [inline component/src/test/java/org/wso2/extension/siddhi/gpl/execution/streamingml/regression/adaptivemodelrules/AdaptiveModelRulesRegressorStreamProcessorExtensionTestcase.java:407]: Need an Assert.fail() here

Comment (tishan89) [reply component/src/test/java/org/wso2/extension/siddhi/gpl/execution/streamingml/regression/adaptivemodelrules/AdaptiveModelRulesRegressorStreamProcessorExtensionTestcase.java:407]: Check same for below tests

File: component/src/test/java/org/wso2/extension/siddhi/gpl/execution/streamingml/regression/adaptivemodelrules/AdaptiveModelRulesRegressorStreamProcessorExtensionTestcase.java (hunk 1 above)
Comment (pcnfernando) [reply component/src/test/java/org/wso2/extension/siddhi/gpl/execution/streamingml/regression/adaptivemodelrules/AdaptiveModelRulesRegressorStreamProcessorExtensionTestcase.java:370]: Updated testcases in [#2d0eaacc4ac535a29697521983533728ba4a6f66](https://github.com/wso2-extensions/siddhi-gpl-execution-streamingml/pull/16/commits/2d0eaacc4ac535a29697521983533728ba4a6f66)

File: component/src/test/java/org/wso2/extension/siddhi/gpl/execution/streamingml/regression/adaptivemodelrules/AdaptiveModelRulesRegressorStreamProcessorExtensionTestcase.java (hunk 2 above)
Comment (pcnfernando) [reply component/src/test/java/org/wso2/extension/siddhi/gpl/execution/streamingml/regression/adaptivemodelrules/AdaptiveModelRulesRegressorStreamProcessorExtensionTestcase.java:407]: Updated testcases in [#2d0eaacc4ac535a29697521983533728ba4a6f66](https://github.com/wso2-extensions/siddhi-gpl-execution-streamingml/pull/16/commits/2d0eaacc4ac535a29697521983533728ba4a6f66)

**Pull request merged**